src/Mod/TechDraw/Gui/QGIViewBalloon.cpp

```diff
@@ -493,29 +492,51 @@ void QGIViewBalloon::balloonLabelDragged(bool ctrl)
     DrawView* balloonParent = getSourceView();
     if (balloonParent != nullptr) {
         scale = balloonParent->getScale();
+        double xPos = Rez::appX(balloonLabel->X() / scale);
```

Comment:
Are these really locals? They look unused.

Reply:
Yes. thanks for spotting, is now fixed